x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicators_flyout/indicators_flyout.tsx

```diff
@@ -107,16 +116,27 @@ export const IndicatorsFlyout: VFC<IndicatorsFlyoutProps> = ({
     prefix: 'simpleFlyoutTitle',
   });
 
+  const titleMessage = useMemo(() => {
+    return selectedTabId === TAB_IDS.overview ? (
+      <FormattedMessage
+        id="xpack.threatIntelligence.indicator.flyout.panelTitleWithOverviewTab"
+        defaultMessage="Indicator Details"
+      />
+    ) : (
+      <FormattedMessage
```

Comment:
I think I like the idea of changing the title to show the name of the indicator, just double-check with @ferenrigue if you haven't already as the designs are different here